include/class.thread.php

```diff
@@ -993,14 +993,19 @@ function lookupByEmailHeaders(&$mailinfo, &$seen=false) {
                 //Lookup by ticket number
                 && ($ticket = Ticket::lookupByNumber($match[1]))
                 //Lookup the user using the email address
-                && ($user = User::lookup(array('emails__address' => $mailinfo['email'])))) {
+                && ($user = User::lookup(array('emails__address' => $mailinfo['email'])))
+                && ($staff = Staff::getIdByEmail($mailinfo['email']))) {
```

Comment:
I missed this comment...  Hrm...  that could definitely be the case..  We have been using osticket for over 3 years now, more then likely all staff members have a user id in there by now...

I'll account for the or as well